src/Entity/ContentMagicTraits.php

```diff
@@ -68,8 +68,8 @@ public function magicEditLink()
     public function magicTitleFields(): array
     {
         // First, see if we have a "title format" in the contenttype.
-        if ($title_format = $this->getDefinition()->get('title_format')) {
-            return (array) $title_format;
+        if ($this->getDefinition()->get('title_format')) {
```

Comment:
sometimes getters can we heavy... can we stick everywhere with convention like:
`$title_format = $this->getDefinition()->get('title_format');`
`if ($title_format)`

Reply:
I've solved it slightly diferently: 

```php
        $definition = $this->getDefinition();

        // First, see if we have a "title format" in the contenttype.
        if ($definition->has('title_format')) {
            return (array) $definition->get('title_format');
        }
```